crates/consensus-logic/src/csm/state_tracker.rs

```diff
@@ -11,7 +11,8 @@ use strata_state::{
 };
 use tracing::*;
 
-use crate::{client_transition, errors::Error};
+use super::client_transition;
+use crate::errors::Error;
```

Comment:
I don't think this change is needed. Also it is not consistent...

Reply:
Again this was from modules that got moved. :P